corev_apu/fpga/Makefile

```diff
@@ -12,7 +12,19 @@ ips := xlnx_axi_clock_converter.xci  \
        xlnx_axi_quad_spi.xci         \
        xlnx_axi_gpio.xci             \
        xlnx_clk_gen.xci              \
-       xlnx_mig_7_ddr3.xci
+	   xlnx_mig_ddr4.xci	     \
+       xlnx_axi_dwidth_converter_512_64.xci
+
+ips_standard := xlnx_axi_clock_converter.xci  \
+       xlnx_axi_dwidth_converter.xci \
+       xlnx_axi_quad_spi.xci         \
+       xlnx_axi_gpio.xci             \
+       xlnx_clk_gen.xci      
+
+ips_ddr3 := xlnx_mig_7_ddr3.xci
+
+ips_ddr4 := xlnx_mig_ddr4.xci	     \
+       		xlnx_axi_dwidth_converter_512_64.xci 
```

Comment:
Are the new variable used ? It looks like you change the default behaviour genesys2, we do not want that.